Library/Formula/cgal.rb

```diff
@@ -51,6 +56,16 @@ def install
     args << '.'
     system "cmake", *args
     system "make install"
+
+    if build.with? "ipelets"
+      cd "demo/CGAL_ipelets"
+      system "cmake . -DCGAL_DIR=../.."
+      system "make"
+      system "make install DESTDIR=#{prefix}"
```

Comment:
Also, maybe work installing this into a location in the `buildpath` and then using `lib.install` from there rather than putting it in the `prefix` and then deleting it.